packages/debs/arm64/Dockerfile

```
@@ -0,0 +1,43 @@
+FROM arm64v8/ubuntu:jammy
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt update -y && \
+    apt install -y checkpolicy curl devscripts equivs g++ gawk gcc git libaudit-dev libauparse0 \
+    libblas3 libcap-ng-dev libgfortran5 liblapack3 libssl-dev make pkg-config policycoreutils policycoreutils-dev \
+    policycoreutils-python-utils python3 python3-audit python3-decorator python3-networkx \
+    python3-numpy python3-selinux python3-semanage python3-sepolgen python3-sepolicy \
+    python3-setools python3-yaml selinux-basics selinux-policy-default selinux-policy-dev \
+    selinux-utils semodule-utils setools sudo tar unzip wget zip && \
+    rm -rf /var/lib/apt/lists/*
+
+ADD https://github.com/Kitware/CMake/releases/download/v3.30.4/cmake-3.30.4-linux-aarch64.sh /tmp/cmake-3.30.4-linux-aarch64.sh
+
+RUN mkdir -p /opt/cmake
+RUN sh /tmp/cmake-3.30.4-linux-aarch64.sh --prefix=/opt/cmake --skip-license
```

We could create a variable for the CMAKE version in case we need to build the package with a different one because the dependencies require it.

```suggestion
ENV CMAKE_VERSION=3.30.4

ADD https://github.com/Kitware/CMake/releases/download/v3.30.4/cmake-$CMAKE_VERSION-linux-aarch64.sh /tmp/cmake-$CMAKE_VERSION-linux-aarch64.sh

RUN mkdir -p /opt/cmake
RUN sh /tmp/cmake-$CMAKE_VERSION-linux-aarch64.sh --prefix=/opt/cmake --skip-license
```